templates/activity.mustache

```diff
@@ -101,9 +107,10 @@
 
     <div id="monassis" class="monassis monassis--practice monassis--maths monassis--siyavula-api">
         <div class="question-wrapper">
+            <div id="sequence-num"></div>
```

Comment:
This might make the spacing look a bit odd it's not a standalone list activity.  Can we only include this div if this is a standalone list activity?